client/app/services/query.js

```diff
@@ -293,6 +294,21 @@ function QueryResource($resource, $http, $q, $location, currentUser, QueryResult
     return this.getParameters().get();
   };
 
+  Query.prototype.getLocalParametersDefs = function getLocalParametersDefs() {
+    if (!this.$localParameters) {
+      this.$localParameters = this.getParametersDefs().filter(p => !p.global);
```

Comment:
Oh I see. The parameters object is cached, but I didn't look into what `get` did. I'll move it to dashboard.